enterprise/server/tasksize/tasksize.go

```diff
@@ -230,6 +238,19 @@ func commandKey(cmd *repb.Command) (string, error) {
 	return fmt.Sprintf("%s/%x", arg0, sha256.Sum256(b)), nil
 }
 
+// Unsized returns a TaskSize indicating that the size is not yet known.
+func Unsized() *scpb.TaskSize {
```

Comment:
i'm a little confused by this Unsized being -1, -1, but the above consts UnsizedTask{MemoryBytes,MilliCPU} being 5600 / 4000. What is the difference